.github/PULL_REQUEST_TEMPLATE/sdk_configuration_template.md

```diff
@@ -0,0 +1,25 @@
+# SDK configuration pull request
+
+## Purpose of this PR
+
+- [ ] Add or update only SDK configuration where there are no changes to the API spec and therefore there is no need for an ARM or Stewardship Board API review.
```

Comment:
Had gpt re-write to make this a bit easier to follow:

```suggestion
- [ ] Make changes to the SDK configuration only when there are no modifications to the API specification, eliminating the need for an ARM or Stewardship Board API review.
```

Reply:
thank you!